ui/forms/MainWindow.qml

```diff
@@ -127,34 +128,48 @@ ApplicationWindow {
     text: `Zoom: ${(systemPreview.zoom * 100).toFixed(0)}%`
   }
 
-  Button {
+  RowLayout {
```

Comment:
Nit: This entire section is more of an ad-hoc layout, so I think a RowLayout is slightly overkill here. I'd use plain anchors. Treat this as an optional change though, this still works fine.

Reply:
That's and easy change, I'll do it in 5 minutes :)